docs/magicc_variables.rst

```diff
@@ -0,0 +1,9 @@
+.. include:: _custom_rst_shortcuts.rst
+
+.. _`MAGICC variables`:
+
+MAGICC variables
+================
+
+A list of the 82 most important variables in MAGICC, a plain-english summary of what they are and a sense of their possible ranges is given in Table S1 of the Supplementary Information of `Meinshausen et al. Nature 2009 <https://www.nature.com.ezp.lib.unimelb.edu.au/articles/nature08017#supplementary-information>`_.
```

Comment:
Great! Can you fix the proxy link? Is the document available without login as well?

Reply:
Can access from here, so should be fine ... but we could use the direct link to the PDF: https://media.nature.com/original/nature-assets/nature/journal/v458/n7242/extref/nature08017-s1.pdf